src/Libraries/RevitNodesUI/RevitTypes.cs

```diff
@@ -122,17 +122,35 @@ public RevisionNumberType(IEnumerable<PortModel> inPorts, IEnumerable<PortModel>
     [NodeDescription("ParameterTypeSelectorDescription", typeof(DSRevitNodesUI.Properties.Resources))]
     [IsDesignScriptCompatible]
     [OutPortTypes("string")]
+    [IsVisibleInDynamoLibrary(false)]
+    [Obsolete("This node may remove in future version. Use SpecTypes instead.")]
     public class ParameterType : CustomGenericEnumerationDropDown
```

Comment:
Hi @QilongTang , I wanted to set this Node to Obsolete, but found that **NodeObsolete** cannot be used outside of methods and properties.
I just make this node - 'Select Parameter Type' invisible in Dynamo Library, and update its NodeDescription.
Do you have any suggestions here?